src/QsCompiler/Tests.Compiler/SyntaxTests.fs

```diff
@@ -38,12 +38,17 @@ let ``String parser tests`` () =
             Assert.Equal(offset + 1, parsed.Length)
             Assert.Equal(char, parsed.[offset])
         | _ -> Assert.True(false, "failed to parse")
-    testChar 0 '\t' "\"\t\""
-    testChar 0 '\n' "\"\r\"" // carriage returns will get replace by \n
-    testChar 0 '\n' "\"\n\""
-    testChar 3 '\t' "$\"{0}\t\""
-    testChar 3 '\n' "$\"{0}\r\"" 
-    testChar 3 '\n' "$\"{0}\n\""
+    testChar 0 '\t' "\"\\t\""
```

Comment:
If you precede the string with a @ character, it will be a verbatim string. That may help with readability here.

Reply:
That's a good tip, I'll try that!

Reply:
Hmm, I may have led you astray. The `@` does indeed prevent escaping, removing the need for double backslashes. But you still need a way to embed the `"` character in strings. I believe F# supports strings delimited with single quotes, so you could do something like `@'"\t"'`.

Reply:
https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/strings#triple-quoted-strings

Reply:
Looks like it doesn't support the single quote strings, but I linked an alternative solution with F#'s triple quote syntax.

Reply:
Yeah, the verbatim strings helped some but the quotes get messy. I'll try the linked syntax.

Reply:
Unfortunately, I don't think triple-quoted strings ends up being that much better:
`testChar 0 '\t' """"\t" """`
I end up needing to have a space between the last quote in the literal string and the triple-quote that ends the string, otherwise the string isn't terminated properly. Any character will work, but something other space starts to look significant. On the other hand, I can't have a space at the beginning because it confuses the stringLiteral parser itself, which expects the string to start with `"`.

Maybe it's years of working on Windows having warped my mind, but I'm partial to going back to traditional string escaping, since my brain has an easier time with this:
`testChar 0 '\t' "\"\\t\""`

Reply:
Yah I think that's probably the right call. The triple quoting is more messy than helpful here, and the `@` doesn't get the right results for nested quotes.